news/2020-02-04-community-contributors-2019.md

```diff
@@ -0,0 +1,212 @@
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/eiri.jpg)](https://osu.ppy.sh/users/3388410)
+
+Everything you'd expect from a beatmap nominator and more, **eiri-** is definitely one of the best around. She's super active and excels in all of nominating, popping, and disqualifying maps, as well as generally checking maps in qualified. In addition to that, she's been very active in pushing maps from mappers both new and old across tons of musical genres. With everything covered, eiri- is definitely good at what she does!
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/bossandy.jpg)](https://osu.ppy.sh/users/360437)
+
+There are those who are incredible simply by being themselves. **bossandy** is not only a good nominator in terms of modding, but he is also very open minded about different mapping styles. He always tries to push maps from all kinds of difficulties, regardless of how unconventional it may be, while still maintain a good proficiency. His great performance paired with his very kind personality is what makes him stand out and earn not only the respect of others, but also the title of Elite Nominator!
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Lasse.jpg)](https://osu.ppy.sh/users/896613)
+
+While also serving as a key member of the QAT and later the NAT, **Lasse** has managed to maintain really high activity in nominating as well. He successfully nominated over 100 maps this year, and also disqualified more than anyone else, all while having a keen eye in his modding. Lasse's mods are some of those that consistently understand what mappers are doing, spotting every mistake, and more!
+
+### osu!taiko
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/komasy.jpg)](https://osu.ppy.sh/users/1980256)
+
+**komasy** made a huge impact on osu!taiko during 2019. Within his 9 month tenure before resigning, komasy helped to push over **100** mapsets to the ranked category! On top of that, he has been very easy to approach, always responding if you have a question or need an opinion. komasy has helped osu!taiko as a mode, helped newer mappers to figure things out, and been a beacon of knowledge in the community.
+
+[![](/wiki/shared/news/2020-02-04-community-contributors-2019/Tyistiana.jpg)](https://osu.ppy.sh/users/1421452)
+
+Having stable activity throughout 2019, **Tyistiana** takes care with each and every mapset. Additionally, he actively partakes in disqualification discussions, and he recently does the vast majority of QAH checks too. He is serving actively both to push content and assure the quality of maps going for ranked, and is truly a backbone for the osu!taiko game mode!
```

Comment:
Imo. this whole paragraph needs some restructuring and simplification to correctly reflect the tone of everything else. Here's my take:

> **Tyistiana** has through stable activity and careful work as a Beatmap Nominator helped further osu!taiko as a game mode. Additionaly, he has actively partaken in disqualification discussions and done work inside the qualified section to assure a high quality of ranked maps. Tyistiana is one who can truly be called a backbone of osu!taiko.

Reply:
also, mentioning the specific role he has done instead of relying on the user to _know_  beforehand would be good.

edit: i changed the above comment to reflect this change